tests/suites/sanity/sanity_test.go

```diff
@@ -90,6 +93,42 @@ var _ = Describe(TestName, func() {
 		assertNumberOfCassandraNodes(NodeCount)
 	})
 
+	It("Updates the instance's cpu and memory", func() {
+		newMemMiB := 3192
+		newMemLimitMiB := 3192
+		newMemBytes := 3347054592
+
+		newCpu := 800
+		newCpuLimit := 1100
+
+		err := Operator.Instance.UpdateParameters(map[string]string{
+			"NODE_MEM_MIB":       strconv.Itoa(newMemMiB),
+			"NODE_MEM_LIMIT_MIB": strconv.Itoa(newMemLimitMiB),
+			"NODE_CPU_MC":        strconv.Itoa(newCpu),
+			"NODE_CPU_LIMIT_MC":  strconv.Itoa(newCpuLimit),
+		})
+		Expect(err).To(BeNil())
+
+		err = Operator.Instance.WaitForPlanInProgress("deploy")
+		Expect(err).To(BeNil())
+
+		err = Operator.Instance.WaitForPlanComplete("deploy")
+		Expect(err).To(BeNil())
+
+		pod, err := kubernetes.GetPod(Client, TestInstance+"-node-0", TestNamespace)
+
+		Expect(err).To(BeNil())
+		Expect(pod).To(Not(BeNil()))
+
+		Expect(pod.Spec.Containers[0].Resources.Requests.Cpu().AsDec().UnscaledBig()).To(Equal(big.NewInt(int64(newCpu))))
+		Expect(pod.Spec.Containers[0].Resources.Requests.Memory().AsDec().UnscaledBig()).To(Equal(big.NewInt(int64(newMemBytes))))
+
+		Expect(pod.Spec.Containers[0].Resources.Limits.Cpu().AsDec().UnscaledBig()).To(Equal(big.NewInt(int64(newCpuLimit))))
+		Expect(pod.Spec.Containers[0].Resources.Limits.Memory().AsDec().UnscaledBig()).To(Equal(big.NewInt(int64(newMemBytes))))
```

Comment:
```suggestion
		Expect(pod.Spec.Containers[0].Resources.Limits.Memory().AsDec().UnscaledBig()).To(Equal(big.NewInt(int64(newMemLimitMiB))))
```

Reply:
That was actually intentional, as we're setting 3192, but the returned value from k8s is (3192 * 1024 * 1024).

I've changed it now so that this is correct and more explicit.